### PR TITLE
feat(ui): implement better getting started ui

### LINKED ISSUE #1553: Better "Getting Started UI"
The `How do you want to connect to the Lightning Network?` screen is not as inviting and intuitive as it could be. Considering this is how users first interact with Zap, I think we should simplify the copy, add some helpful icons and maybe re-design the layout if need be.

Below is the current screen in `next`:

screen shot 2019-02-10 at 12 37 46 pm

I think the title of the options should be one word: `Create`, `Connect` and `Recover`. Simpler UX. I also think each should have it's own icon maybe and we could play around with new layout ideas and see if any feel good

Each title option has its own icon and below is the layout design. 

screen shot 2019-03-05 at 1 20 45 pm
screen shot 2019-03-05 at 1 21 13 pm
screen shot 2019-03-05 at 2 33 50 pm

https://app.zeplin.io/project/5bacdbad33af886d419f1b34/screen/5c7ea7efbc2e8abcea0eb7a5

https://app.zeplin.io/project/5bacdbad33af886d419f1b34/screen/5c7ea7edbf0d81bce4546b39

https://app.zeplin.io/project/5bacdbad33af886d419f1b34/screen/5c7ea7eed693e3bc7283dc38

### PR DESCRIPTION
**Pull request by korhaliv**
## Description:
Resolves #1553 

## How Has This Been Tested?
Manually

## Screenshots (if appropriate):
image

## Types of changes:
UI improvement

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

**Comment by korhaliv** [comment]
@mrfelton @JimmyMow added subtle fade-in entrance for the bolt.

**Comment by korhaliv** [comment]
@JimmyMow updated
image

File: app/components/Onboarding/Steps/ConnectionTypeItem.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import { Card as BaseCard, Radio, Text, Heading } from 'components/UI'
+
+import ConnectOnboarding from 'components/Icon/ConnectOnboarding'
+import ImportOnboarding from 'components/Icon/ImportOnboarding'
+import PlusOnboarding from 'components/Icon/PlusOnboarding'
+import BoltOnboarding from 'components/Icon/BoltOnboarding'
+
+import { animated, Transition } from 'react-spring'
+
+const Box = styled(BaseBox)(opacity)
```
Comment (mrfelton) [inline app/components/Onboarding/Steps/ConnectionTypeItem.js]: Outside of the scope of this PR, but we might want to think about creating our own `Box` component that extends @rebass.Box and adds support for some of the other props that we commonly use, such as opacity and height.

Comment (korhaliv) [reply app/components/Onboarding/Steps/ConnectionTypeItem.js]: You are right, but actually that code wasn't even needed in the current layout design. So I removed it

**Comment by korhaliv** [comment]
@mrfelton rebased & centered the layout

**Pull request merged**